amlb/runners/docker.py

```diff
@@ -152,6 +159,9 @@ def _generate_script(self, custom_commands):
 ADD . /bench/
 
 RUN (grep -v '^\\s*#' | xargs -L 1 $PIP install --no-cache-dir) < requirements.txt
+RUN chown -R {username}.{username} /bench
+RUN chown -R {username}.{username} /home/{username}
```

Comment:
```suggestion
RUN chown -R {username}:{username} /bench
RUN chown -R {username}:{username} /home/{username}
```

`.` is a deprecated syntax